_src/_includes/themes/apache/home.html

```diff
@@ -75,24 +73,71 @@
+    </div>
+    <div class="nb-card">
+        <h3>Linear Regression Algorithms Demo</h3>
+        This notebook shows:
+          <ul>
+            <li>Install SystemML Python package and jar file</li>
+            <ul>
+              <li>pip</li>
+              <li>SystemML 'Hello World'</li>
+            </ul>
+          </ul>
+        <a class="nb-link" href="https://github.com/apache/incubator-systemml/blob/master/samples/jupyter-notebooks/Linear_Regression_Algorithms_Demo.ipynb" target="_blank"><span class="icon jupyter-logo"></span><span>View on Github</span></a>
+    </div>
+    <div class="nb-card">
+        <h3>SystemML PySpark Recommendation Demo</h3>
+        <p>This demonstrates using SystemML for product recommendationg using Poisson NonNegative Matrix Factorization (PNMF) with PNMF algorithm written using R like syntax.</p>
+        <a class="nb-link" href="https://github.com/apache/incubator-systemml/blob/master/samples/jupyter-notebooks/SystemML-PySpark-Recommendation-Demo.ipynb" target="_blank"><span class="icon jupyter-logo"></span><span>View on Github</span></a>
+    </div>
+    <div class="nb-card">
+        <h3>SystemML Scala Tutorial</h3>
+        <p>This tutorial includes simple example to run DML script and display output.</p>
+        <a class="nb-link" href="https://github.com/apache/incubator-systemml/blob/master/samples/jupyter-notebooks/tutorial1.ipynb" target="_blank"><span class="icon jupyter-logo"></span><span>View on Github</span></a>
+    </div>
+    <div class="nb-card">
+        <h3>SystemML LinearRegCG</h3>
+        <p>SystemML Linear Regression in Zeppelin Notebook.</p>
+        <a class="nb-link" href="https://github.com/apache/incubator-systemml/blob/master/samples/zeppelin-notebooks/SystemML_LinearRegCG.json" target="_blank"><span class="icon zeppelin-logo"></span><span>View on Github</span></a>
+    </div>
+    <div class="nb-card">
+        <h3>SystemML Zeppelin Tutorial</h3>
+        <p>SystemML Zeppelin tutorial.</p>
+        <a class="nb-link" href="https://github.com/apache/incubator-systemml/blob/master/samples/zeppelin-notebooks/tutorial1_zeppelin.json" target="_blank"><span class="icon zeppelin-logo"></span><span>View on Github</span></a>
+    </div>
+
+  </div>
+
 </section>
 
 <!-- Contact Us -->
 <section class="full-stripe">
   <div class="ml-container ml-container--horizontally-center">
     <div class="col col-12 content-group ">
-      <h2>Subscribe to Our Mailing Lists</h2>
-      <p>Subscribe to our development mailing list for SystemML updates and news. Once subscribed, <a href="mailto:{{ site.data.project.dev_list }}">join the conversation</a>.
-      As SystemML grows, so will our community. Check out <a href="community.html#mailing-list">All Mailing Lists</a>.
+      <h2>SContribute</h2>
+      <p>Contribute to Apache SystemMLTM by subscribing to our developer mailing list for updates and news. Check out <a href="community.html#mailing-list">All Mailing Lists</a>.
```

Comment:
I think {{ site.data.project.name }} can be used if you want to include TM as a superscript. Otherwise I think it's probably fine not to have the TM